src/lithium/__init__.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# coding=utf-8
+# flake8: noqa
+# pylint: disable=missing-docstring
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import
+
+from .interestingness import crashes
+# from .interestingness import diff_test  # diff_test is not used outside of Lithium
+from .interestingness import env_vars
+from .interestingness import file_ingredients
+from .interestingness import hangs
+from .interestingness import outputs
+from .interestingness import range  # pylint: disable=redefined-builtin
+from .interestingness import timed_run
+from .interestingness import ximport
```

Comment:
I don't think we need any of these in the `lithium` namespace. They should be in `lithium.interestingness`